.pipelines/v2/templates/job-build-project.yml

```diff
@@ -376,6 +376,12 @@ jobs:
         msbuildArchitecture: x64
         maximumCpuCount: true
 
+  ### HACK: On ARM64 builds, building an app with Windows App SDK copies the x64 WebView2 dll instead of the ARM64 one. This command makes sure the right dll is used.
+  - script: |
+      COPY /Y "packages\Microsoft.Web.WebView2.1.0.2739.15\runtimes\win-ARM64\native_uap\Microsoft.Web.WebView2.Core.dll" "ARM64\Release\WinUI3Apps\Microsoft.Web.WebView2.Core.dll" || exit /b 1
+    displayName: HACK Copy core WebView2 ARM64 dll to output directory
+    condition: eq(variables['BuildPlatform'],'arm64')
```

Comment:
FWIW I prefer template conditions (`${{if...`) rather than runtime ones, but this is fine.

Reply:
I did try adding that, but from what I could see, BuildPlatform is only defined at runtime, not compile when it's generated. If it's fine with you, I'd prefer to expedite this one  as it is ;)